docs/API/core/analytics.md

```diff
@@ -256,3 +256,92 @@ Fliplet.Apps.Analytics.count(appId, { group: [ 'data._userEmail' ] }).then(funct
   // console.log(count)
 });
 ```
+
+### Fetch aggregated analytics
+
+You can also use JS APIs to obtain pre-aggregated analytics data for your apps.
+
+<p class="quote"><strong>Note</strong>: this JS API is only available when app users have logged in with the <strong>Fliplet Studio</strong> component or when using <strong>Fliplet Viewer</strong>.</p>
+
+Here are some of the use cases for fetching pre aggregated data
+
+#### Fetch aggregated data for the app
+
+```js
+Fliplet.Apps.Analytics.Aggregate.get({
+    "source": "production",
+    "from": "2024-05-01",
+    "to": "2024-05-31",
+    "group":"app",
+    "sum": ['sentEmails', 'sentSMS', 'sentPushNotifications', 'totalDevices', 'totalSessionDuration'],
+}).then(function (data) {
+  // console.log(data)
+});
+```
+- `source`: can be either `production` or `master`. production will be used to fetch data for the live app
```

Comment:
"Production" Should be capital

Reply:
@debankurghosh2061 We use camel case for property and variable names in our API. Therefore, they need to be passed in this format.

Reply:
In that case, I would put 'production' in the quotes as well like you have in the first sentence.